modules/nf-core/pymygene/templates/pymygene.py

```diff
@@ -0,0 +1,215 @@
+#!/usr/bin/env python3
```

Comment:
Needs a licence, and credit for any other authors (unless written from scratch by yourself). For example see https://github.com/nf-core/modules/blob/cabcc0dadf8366aa7a9930066a7b3dd90d9825d5/modules/nf-core/custom/catadditionalfasta/templates/fasta2gtf.py#L3.

Reply:
The script itself is written only by me. Then do i need to mention the original authors of the mygene package there, or is it enough to have it mentioned in the meta file?

Reply:
Just your name then- but choose a license.